src/auth/dto/create-user.dto.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  IsString,
+  MinLength,
+  Matches,
+  IsNotEmpty,
+  IsEmail,
+  Length,
+} from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+export class CreateUserDto {
+  @IsString()
```

Comment:
PullProofAI

Consider adding a validation rule for the maximum length of the `name` field for consistency.
```diff
+   @MaxLength(50, { message: 'Name should not exceed 50 characters' })
```



<!-- This is an auto-generated comment by PullProofAI -->